### ISSUE TITLE
Mod Menu Settings Game Crash 1.20.2

### ISSUE THREAD

**Issue by nco2k** [KxmischesDomi/MoreBannerFeatures#29]
opening up mod menu and clicking on the settings button of this mod makes the game crash with following error:

- The game crashed whilst rendering screen
- Error: java.lang.NoSuchMethodError: 'void de.kxmischesdomi.morebannerfeatures.renderer.screen.MBFOptionsScreen.method_25420(net.minecraft.class_332)'
- Exit Code: -1073740791

im using the latest files:

- fabric-api-0.90.7+1.20.2.jar
- modmenu-8.0.0.jar
- morebannerfeatures-1.2.0-1.20.jar

**Comment by DomiIRL** [comment]
Yes I also noticed this issue with my other mods that use the same settings screen. Gonna look into it soon.
